GSA-TTS/FAC
# 3718 address historical audits with missing planned actions

## Description
Some historical reports have an empty `corrective action plan` text field. This field is mandatory and causes the migration to fail for those reports. This PR replaces empty `corrective action plan` text fields with `GSA_MIGRATION` keyword.

As of 05/07/2024, I updated the PR to include a similar fix for missing `text of finding`. See #3804 

## Test
- To test this PR, select an historical report and replace one of the `ELECCAPTEXT.TEXT` record with empty string.
- Try to migrate this report
- Confirm `GSA_MIGRATION` is in the corresponding `CapText.planned_action` field in dissemination. 
- Repeat the previous steps against `ELECFINDINGSTEXT.TEXT` to test the fix for missing `text of finding`

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
